algo

```diff
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
```

Comment:
A better way to fix this might be switching to prompting for the user cert password within ansible with the `private: yes` option set instead of reading it from the shell directly.